library/common/config_template.cc

```diff
@@ -207,6 +207,13 @@ stats_flush_interval: {{ stats_flush_interval_seconds }}s
       grpc_service:
         envoy_grpc:
           cluster_name: stats
+  - name: envoy.stat_sinks.metrics_service.mobile
```

Comment:
I am going to put up a PR today to reduce some of the stats we are emitting (and no longer need), as this config effectively doubles the amount of stats emitted